lib/spark.js

```diff
@@ -1,14 +1,15 @@
 /* global Zinnia */
 
 import { ActivityState } from './activity-state.js'
-import { SPARK_VERSION, DELAY_BETWEEN_RETRIEVALS, MAX_CAR_SIZE } from './constants.js'
+import { SPARK_VERSION, MAX_CAR_SIZE, APPROX_ROUND_LENGTH_IN_MS } from './constants.js'
 import { encodeHex } from './deno-encoding-hex.js'
 
 const sleep = dt => new Promise(resolve => setTimeout(resolve, dt))
 
 export default class Spark {
   #fetch
   #activity = new ActivityState()
+  #maxTasksPerNode = 360
```

Comment:
```suggestion
  #maxTasksPerNode
```

instead of assigning a default that could be vastly off, what about not using the variable until we have fetched a value?

Reply:
What if the first iteration fails to fetch the value (e.g. because we are offline)? We need to know the delay before we start the second iteration. See https://github.com/filecoin-station/spark/pull/47/files#diff-9eb2d17c6352497045e4a22df4b4957d5d44f3cfff0c66c6519d07705ee1ed86R177